lib/simple_lru.js

```diff
@@ -43,10 +44,18 @@ Cache.prototype.get = function(key,hit){
      * it to the head of linked list  
      */
     hit = hit != undefined && hit != null ? hit : true;
-    if(cacheVal && hit)
+
+    /*
+    * Define if maxTime is used and if it's valid.
+    */
+    var cacheTimeValid = (cacheVal && cacheVal.time == undefined) || 
+                         (cacheVal && Date.now() - cacheVal.time < this.maxTime);
+
+    if(cacheVal && hit && cacheTimeValid) 
```

Comment:
These 3 lines can be condensed into one with fewer conditionals.

`if(cacheVal && (!cacheVal.time || (Date.now() - cacheVal.time) < this.maxTime))`